### ISSUE TITLE
Add bridge from EqualTo to Interval

### ISSUE THREAD

**Issue by blegat** [jump-dev/MathOptInterface.jl#852]
Many solvers (e.g. Cbc, Clp) don't really support `F`-in-`EqualTo`, they simply transform it into `F`-in-`Interval` with both lower and upper bounds equal to the same value.
They should therefore not support this constraint and we should add a bridge for this.

**Comment by blegat** [comment]
Another application is for for allowing `FileFormats.SDPA.Model` to support `ScalarAffineFunction`-in-`EqualTo`.

**Comment by blegat** [comment]
> Another application is for for allowing `FileFormats.SDPA.Model` to support `ScalarAffineFunction`-in-`EqualTo`.

This is resolved by https://github.com/JuliaOpt/MathOptInterface.jl/pull/1005

**Comment by odow** [comment]
This was fixed by #1005

```Julia
using MathOptInterface
const MOI = MathOptInterface
MOI.Utilities.@model(
    Model,
    (),
    (MOI.GreaterThan, MOI.LessThan),
    (),
    (),
    (),
    (MOI.ScalarAffineFunction,),
    (),
    ()
)
MOI.supports_constraint(::Model{T}, ::Type{MOI.SingleVariable}, ::Type{MOI.EqualTo{T}}) where {T} = false

julia> model = MOI.Bridges.full_bridge_optimizer(Model{Float64}(), Float64)
MOIB.LazyBridgeOptimizer{MOIU.GenericModel{Float64, MOIU.ObjectiveContainer{Float64}, MOIU.VariablesContainer{Float64}, ModelScalarConstraints{Float64, MOIU.VectorOfConstraints{MOI.ScalarAffineFunction{Float64}, MOI.GreaterThan{Float64}}, MOIU.VectorOfConstraints{MOI.ScalarAffineFunction{Float64}, MOI.LessThan{Float64}}}}}
with 0 variable bridges
with 0 constraint bridges
with 0 objective bridges
with inner model MOIU.GenericModel{Float64, MOIU.ObjectiveContainer{Float64}, MOIU.VariablesContainer{Float64}, ModelScalarConstraints{Float64, MOIU.VectorOfConstraints{MOI.ScalarAffineFunction{Float64}, MOI.GreaterThan{Float64}}, MOIU.VectorOfConstraints{MOI.ScalarAffineFunction{Float64}, MOI.LessThan{Float64}}}}

julia> x = MOI.add_variable(model)
MathOptInterface.VariableIndex(1)

julia> MOI.add_constraint(model, x, MOI.EqualTo(1.0))
MathOptInterface.ConstraintIndex{MathOptInterface.SingleVariable, MathOptInterface.EqualTo{Float64}}(1)

julia> print(model)
Feasibility

Subject to:

SingleVariable-in-EqualTo{Float64}
 v[1] == 1.0

julia> print(model.model)
Feasibility

Subject to:

SingleVariable-in-GreaterThan{Float64}
 v[1] >= 1.0

SingleVariable-in-LessThan{Float64}
 v[1] <= 1.0
 ```
